src/Tippy.Util/Hex.cs

```diff
@@ -22,5 +23,19 @@ public static BigInteger HexToBigInteger(string hex)
         }
 
         public static string UInt64ToHex(UInt64 num) => $"0x{num:x}";
+
+        public static byte[] HexToBytes(string i)

```

Comment:
Does this convert hex format number, or just hex string (like private key) to bytes? If it's for string, not for number, we can use `System.Convert`'s `FromHexString` and `ToHexString`.

Reply:
Yes, it's for string, and updated now.